.vscode/settings.json

```diff
@@ -1,15 +1,19 @@
 {
     "editor.tabSize": 2,
     "cSpell.words": [
+        "altnet",
+        "Clawback",
         "hostid",
         "keypair",
         "keypairs",
         "multisign",
         "multisigned",
         "multisigning",
         "preauthorization",
+        "rippletest",
```

Comment:
Is this just for the url in the snippet?

Reply:
("altnet" also doesn't seem to be used anywhere)

Reply:
Yeah, just the snippets (I just wanted the squigglies to go away, this file doesn't really matter much - the VSCode extension is helpful for finding typos though)